Framework/DataHandling/test/SaveANSTOAsciiTest.h

```diff
@@ -64,9 +67,9 @@ class SaveANSTOAsciiTest : public CxxTest::TestSuite {
     boost::split(columns, fullline, boost::is_any_of("\t"),
                  boost::token_compress_on);
     TS_ASSERT_EQUALS(columns.size(), 4);
-    TS_ASSERT_DELTA(boost::lexical_cast<double>(columns.at(0)), 1.5, 0.01);
-    TS_ASSERT_DELTA(boost::lexical_cast<double>(columns.at(1)), 1, 0.01);
-    TS_ASSERT_DELTA(boost::lexical_cast<double>(columns.at(2)), 1, 0.01);
+    TS_ASSERT_DELTA(boost::lexical_cast<double>(columns.at(0)), 1., 0.01);
```

Comment:
Why is this not 1.5 anymore?

Reply:
Because the test data has changed from histogram to point data. In `createWS` below, originally one
- created a histogram workspace with 10 bins, 11 bin edges
- set the bin edges to a `vector` of 10 elements (`m_dataX` / `m_data0`)

I took this as a hint that the original intention was to use point data for testing. Now I am not sure anymore. I'll change this back such that the assertions don't need to be changed. Same applies to the other save ASCII algorithms. Luckily, they are deprecated and will all be gone in a future release.